### PR TITLE
A name-based invariant about Lowercase

### PR DESCRIPTION
**Pull request by eggrobin**
See https://github.com/unicode-org/unicodetools/pull/708#discussion_r1522375203.

### CODE REVIEW

**Comment by eggrobin** [comment]
Wait no, I also need to catch LETTER SMALL, which is what we have in #708.

**Comment by markusicu** [comment]
please say when ready to look

**Comment by eggrobin** [comment]
Should be fine now.

File: unicodetools/src/main/resources/org/unicode/text/UCD/UnicodeInvariantTest.txt [hunk 1]
```diff
@@ -586,12 +586,22 @@ Let $nonAlphabeticAvagrahas = [\N{TIBETAN MARK PALUTA}]  # A punctuation mark.
...
 
 # Combining letters are often alphabetic (medievalist abbreviations).
 # The others are diacritic (cantillation marks, phonetics).
 # See 177-C52.
 \p{name=/COMBINING .* LETTER/} ⊆ [\p{Alphabetic}\p{Diacritic}]
 
+## Consistency of Lowercase with decompositions.
+# Note that the same is not true of Uppercase.
+# A non-lowercase character has non-lowercase characters in its decomposition,
+# or it its decomposition is <square> (㋍ etc.).
```
Comment (markusicu) [inline unicodetools/src/main/resources/org/unicode/text/UCD/UnicodeInvariantTest.txt:602]:
```suggestion
# or its decomposition is <square> (㋍ etc.).
```

**Review by markusicu — APPROVED** [review]

**Pull request merged**